msleep is now also in linux-2.4 (since 2.4.29-rc1). So there's no need for it at all.
	--jsaw

--- alsa-driver-1.0.7/include/adriver.h~	2004-11-10 11:12:20.000000000 +0100
+++ alsa-driver-1.0.7/include/adriver.h	2005-01-09 23:28:00.613631272 +0100
@@ -527,15 +527,6 @@
 static inline void class_simple_device_remove(int devnum) { return; }
 #endif
 
-/* msleep */
-#if LINUX_VERSION_CODE < KERNEL_VERSION(2, 6, 6)
-#define msleep(msecs) \
-	do { \
-		set_current_state(TASK_UNINTERRUPTIBLE); \
-		schedule_timeout(((msecs) * HZ + 999) / 1000);	\
-	} while (0)
-#endif
-
 #if LINUX_VERSION_CODE < KERNEL_VERSION(2, 6, 0)
 #define snd_card_set_dev(card,dev) /* no struct device */
 #endif
